webapp/channels/src/components/suggestion/at_mention_provider/at_mention_suggestion.tsx

```diff
@@ -33,169 +31,167 @@ interface Group extends Item {
     member_count: number;
 }
 
-class AtMentionSuggestion extends Suggestion {
-    render() {
-        const {intl} = this.props;
-        const {isSelection, item} = this.props;
-
-        let itemname: string;
-        let description: ReactNode;
-        let icon: JSX.Element;
-        let customStatus: ReactNode;
-        if (item.username === 'all') {
-            itemname = 'all';
-            description = (
-                <FormattedMessage
-                    id='suggestion.mention.all'
-                    defaultMessage='Notifies everyone in this channel'
+const AtMentionSuggestion = React.forwardRef<HTMLDivElement, SuggestionProps<Item>>((props, ref) => {
```

Comment:
These diffs might be a bit easier to read if you turn on this setting to hide changes to leading whitespace
![Screen Shot 2023-05-18 at 5 00 46 PM](https://github.com/mattermost/mattermost-server/assets/3277310/b3c11920-d792-4a00-8bea-38e3fd77c100)